src/deriver/DeriverUtils.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let jose = require('node-jose');
+
+/**
+ * Scoped key deriver utilities
+ * @module deriver-DeriverUtils
+ * @private
+ */
+class DeriverUtils {
+  /**
+   * @method encryptBundle
+   * @param {string} appPublicKeyJwk
+   * @param {object} bundle
+   * @returns {Promise}
+   */
+  encryptBundle(appPublicKeyJwk, bundle) {
+    bundle = jose.util.asBuffer(bundle);
+
+    return jose.JWK.asKey(appPublicKeyJwk)
```

Comment:
How much validation will this do on the provided JWK?  In [1] we talk a lot about trying to validate the the relier has done the right thing, so I wonder if there are additional checks we should do here, such as:

* Checking that it's an ECDH key
* Checking that it's on curve P-256
* Checking that they haven't accidentally sent us the private key component

It's probably worth us adding some testcases the ensure we catch and reject such erroneous keys.

[1] https://docs.google.com/document/d/1IvQJFEBFz0PnL4uVlIvt8fBS_IPwSK-avK0BRIHucxQ/edit#heading=h.i14db5iwaq97